main/java/com/github/onsdigital/dpjavadurationparse/ParseDuration.java

```diff
@@ -0,0 +1,75 @@
+package com.github.onsdigital.dpjavadurationparse;
+
+import java.time.Duration;
+import java.time.format.DateTimeParseException;
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
Isn't this file duplicated? It seems to be in the `src` folder too so this one could go

Reply:
Thanks Rafa.  Weirdly missed this.  The main and test directories are not shown on my Intellij Project tab weirdly.  Removed them now thanks.